### PR TITLE
Removes Meme/Reference AI Names from Random Name Lists

### PR DESCRIPTION
**Pull request by Sylphet**
## About The Pull Request

does what it says on the tin

## Why It's Good For The Game

Some of the random names are against our rules, yet you can get them from clicking random name.

> Your character must not be a copy or direct reference to an existing character in media or a real person.

This misleads players into thinking the names are allowed and we have to bwoink to get them changed. Basically - entrapment by SS13's code itself!

## Proof Of Testing

Little hard to test something like this for certain, but after a minute of clicking through random AI name, I saw none of the cut names. It's probably fine™

## Changelog

:cl:
del: cut some of the AI names
/:cl:

### CODE REVIEW

File: strings/names/ai.txt [hunk 1]
```diff
@@ -11,20 +11,17 @@ AMEE
 AmigoBot
 Android
 Aniel
-Asimov
```
Comment (Majkl-J) [inline strings/names/ai.txt:14]: This one is included IC and would make sense for an AI. Heck, his lawset is present

**Comment by YakumoChen** [comment]
I'm pretty sure almost all AI names are a reference, this seems like a rather pointless PR.

**Comment by YakumoChen** [comment]
You haven't even removed all the super well-known ones, but I could probably spend all day linking every entry to some notable work of fiction

**Comment by IgiariValkyr** [comment]
> There's a huge difference between a reference to a lesser known work of fiction that all of five people will understand, and **IT IS ME, YOUR STATION AI, R2-D2 FROM THE HIT 1977 MOVIE STAR WARS**.
> 
> Doesn't mean that we have to 1984 every reference to anything ever from the code, or that cutting the worst offenders from the list means we have to ban people without thinking because **oh no, reference detected!** It's as simple as that we probably shouldn't have the game literally suggest people to use obvious, bannable names.

Oh no. Oh no they're not in the random list anymore.
Guess it's time to actually make use of being able to type in names manually to still have my funny meme name.
GigaChad

**Comment by Sylphet** [comment]
based, tbh

**Comment by Swiftfeather** [comment]
> **IT IS ME, YOUR STATION AI, R2-D2 FROM THE HIT 1977 MOVIE STAR WARS**.

my new AI gimmick has been located.

**Comment by YakumoChen** [comment]
> There's a huge difference between a reference to a lesser known work of fiction that all of five people will understand, and **IT IS ME, YOUR STATION AI, R2-D2 FROM THE HIT 1977 MOVIE STAR WARS**.
> 
> Doesn't mean that we have to 1984 every reference to anything ever from the code, or that cutting the worst offenders from the list means we have to ban people without thinking because **oh no, reference detected!** It's as simple as that we probably shouldn't have the game literally suggest people to use obvious, bannable names.

This is a clear double standard, and using the most egregious and hyperbolic case as an argument

**Comment by projectkepler-ru** [comment]
Please remember what I had pinned, the ACEDICK principle
you either double down on all the meme name an AI could get or you don't

**Comment by Elli-Skala** [comment]
No one has mentioned it yet but this removes the name "tik-tok"
And after going onto tg and looking through the blame was added *at least* 14 years ago! It's 100% a modern day lens to like at something diffrent then

What I'm trying to say is: this is cancle culture

**Comment by AllCorruptingWyrm** [comment]
How is this cancel culture.

**Comment by FoxxyRenamon** [comment]
Yeah I personally don't see an issue with these names. It's already against the rules to run with a name that explicitly references real people or popular culture characters, like Bender, or KITT. I don't see why it's needed to do this.

**Comment by ReturnToZender** [comment]
No one really noticed these existed, and no one would notice if they're gone. Was asked for feedback with an admin ping.

This is mostly a virtue-signalling PR like "hey guys, we're working on having RP" and you know what? That's cool.
As someone who was working on replacing all of the shit and memey ERT names that randomly rolled, I totally agree with this PR. May not agree with individual names but no AI uses randomgen names anyway.

My only suggestion: Add some silly new names to keep the variety up?

**Comment by Sylphet** [comment]
> No one really noticed these existed, and no one would notice if they're gone. Was asked for feedback with an admin ping.
> 
> This is mostly a virtue-signalling PR like "hey guys, we're working on having RP" and you know what? That's cool. As someone who was working on replacing all of the shit and memey ERT names that randomly rolled, I totally agree with this PR. May not agree with individual names but no AI uses randomgen names anyway. This removes twelve out of 147 names, so it doesn't really have an impact on the game at all.
> 
> My only suggestion: Add some silly new names to keep the variety up?

See, that's the fun part, the thing I talked about happening, did happen. I wouldn't waste my time if this was a virtue signaling PR.

**Comment by Elli-Skala** [comment]
> How is this cancel culture.

Tik-Tok _isn't_ a refrance it was added in ~2012
It is being canceled for something it didn't do because of the millional coders

**Comment by AllCorruptingWyrm** [comment]
> > How is this cancel culture.
> 
> Tik-Tok _isn't_ a refrance it was added in ~2012 It is being canceled for something it didn't do because of the millional coders

Oh okay, fair enough.

**Comment by Majkl-J** [comment]
This is not needed. Bwoinking someone with "Please change your name" isn't exactly hard, and most of the times it is the ai ahelping "Change my name please I forgot" anyhow.

**Pull request closed without merging**